api/test/wfl/tools/workloads.clj

```diff
@@ -127,14 +127,30 @@
 
 (defn covid-workload-request
   "Make a COVID Sarscov2IlluminaFull workload creation request."
-  [dataset method-configuration workspace]
-  {:pipeline covid/pipeline
-   :source {:name "Terra DataRepo",
-            :dataset dataset}
-   :executor {:name "Terra",
-              :method_configuration method-configuration}
-   :sink {:name "Terra Workspace",
-          :workspace workspace}})
+  [source executor sink]
+  {:source   (merge
+              {:name    "Terra DataRepo",
```

Comment:
You give it a `:name` and it returns a `:type`. Something about this feels odd. I like the separation between the representation in the request and the actual workload data model, but if we just used `:type` I think it would be easier to use.